## Professional Summary:
Computational biologist with 9+ years of experience in **multi-omics analysis**, specializing in **single-cell, spatial, TCR, epitope, and bioinformatics pipeline development**. Experienced in developing single cell, spatial and bulk multi-omics bioinformatics pipelines in neuro-oncology, immune ageing, autoimmunity, population genomics, & rare disease. Seeking to contribute my skills in biotech for oncology, biomarker discovery, or therapeutic target identification.

## Key Skills
•	**Single-Cell:** scRNA, scTCR, CITESeq, scATAC, Multiome, BEAM-T, TetTCR <br>
•	**Spatial:** Xenium, Visium, MERFISH, Cyclic IF, CODEX <br>
•	**Bulk:** Whole Genome Sequencing, Transcriptomics, Epigenomics <br>
•	**Workflow Optimization:** Nextflow, Docker, Singularity & RShiny <br>
•	**Programming:** R, Python, Shell scripting; HPC (Slurm, SGE, LSF) <br>
•	**Machine learning:** scikit-learn, PyTorch <br>
•	**Cloud Computing:** AWS, MS-Azure, GCP <br>
•	**Collaboration:** Extensively worked with clinicians, wet-lab, dry-lab <br>
•	**Industry Consultant:** Consultant for a startup Genomiki Solutions <br>
•	**Academic Mentorship:** Guided postdoc and PhD students <br>
•	**Leadership:** Lead computational team for India 1000 Genome Project <br>
•	**Version Control:** Git, Github

## Professional Experience
**Computational Biologist, VoyantBio, San Francisco** May'25-Now<br>
•	Integrated **pan cancer immune** single cell and spatial dataset to identify the biomarkers to improve **immunotherapy** response.<br>

**Post-Doctoral Fellow, UCSF, San Francisco** Jun'24-Aug'25<br>
•	Analyzed snRNA, ATAC, and spatial (Visium & Xenium) transcriptomics to identify **therapeutic targets in SHH medulloblastoma**<br>
•	Developing **lineage-tracing methodologies** to track tumor evolution using CRISPR barcodes<br>
•	**Foundation model** (scGPT) fine-tuned using >100 paired scRNA **brain cancer metastasis**<br>

**Post-Doctoral Fellow, Mayo Clinic, Rochester** Aug'21–May'24<br> 
(Currently Collaborator)			        
•	Characterized antigen-specific T-cell in **aging and vaccine** using CITESeq, BEAM-T & ATACseq<br>
•	Performed scRNA, scmultiome,scTCR and bulk RNA to identify clonal diversity in naïve CD4 T cells of **Aortitis patients**<br>
•	Integrated single-cell and spatial (Visium, CODEX, and multiplexed Cycif) to study immune cell crosstalk in **autoimmune disorders**<br>

**Graduate Research Fellow CSIR-IGIB, New Delhi** July'16 – July'21<br>
•	Led bioinformatics analysis team for **1,000+ Indian Genome Sequencing Project**, focusing on rare disease variants<br>
•	Developed workflows for **SARS-CoV-2 variant** tracking using Illumina and Nanopore sequencing<br>
•	Variant prioritization in **~300 primary immunodeficiency disorder** patients using genome sequencing<br><br>

## Selected Publications:
Full list [Google Scholar](https://scholar.google.com/citations?user=sCGcKlsAAAAJ&hl=en)<br>
•	Sturmlechner I#, **Jain A#**, et al. Aging trajectories of memory CD8+ T cells differ by their antigen specificity. **Nature Communications (2025)**<br>
•	Sato Y, **Jain A**, et al. Stem-like CD4 T cells in perivascular tertiary lymphoid structures sustain autoimmune vasculitis. **Science Translation Medicine (2023)**<br>
•	**Jain A#**, Bhoyar R#, et al. IndiGenomes: a comprehensive resource of genetic variants from over 1000 Indian genomes. **Nucleic Acid Research (2021)**<br>
•	Zhang H#, Okuyama H#, **Jain A#**, PREX1 improves homeostatic proliferation to maintain a naive CD4+ T cell compartment in older age. **JCI Insight (2024)**<br>
#-co-first author<br>

